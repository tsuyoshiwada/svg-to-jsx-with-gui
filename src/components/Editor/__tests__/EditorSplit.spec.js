import React from 'react';
import { shallow } from 'enzyme';
import EditorSplit from '../EditorSplit';


describe('components#<EditorSplit />', () => {
  test('Should be render', () => {
    const wrapper = shallow(<EditorSplit />);
    expect(wrapper).toMatchSnapshot();
  });
});
