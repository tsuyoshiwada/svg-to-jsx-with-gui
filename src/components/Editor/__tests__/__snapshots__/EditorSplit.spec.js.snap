// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components#<EditorSplit /> Should be render 1`] = `
<div
  className="sc-bdVaJa PAoBY"
/>
`;
